iam/src/index.ts

```diff
@@ -20,11 +20,15 @@ import {
   ChallengeRequestBody,
   VerifyRequestBody,
   CredentialResponseBody,
-} from "@dpopp/types";
+} from "@gitcoin/passport-types";
 
 // ---- Generate & Verify methods
 import * as DIDKit from "@spruceid/didkit-wasm-node";
-import { issueChallengeCredential, issueHashedCredential, verifyCredential } from "@dpopp/identity/dist/commonjs";
+import {
+  issueChallengeCredential,
+  issueHashedCredential,
+  verifyCredential,
+} from "@gitcoin/passport-identity/dist/commonjs";
```

Comment:
```suggestion
} from "@gitcoin/passport-identity/dist/commonjs/src/credentials";
```
Should this import also be updated?

Reply:
updating this import to `commonjs/src/credentials` here and in the test file worked on a fresh git clone 👍 , commit amended